packages/typography/src/components/Code.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledCode } from '../styled';
+
+const SIZE = {
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  LARGE: 'large'
+};
+
+const HUE = {
+  GREY: 'grey',
+  RED: 'red',
+  GREEN: 'green',
+  YELLOW: 'yellow'
+};
+
+/**
+ * Accepts all `code` props
+ */
+const Code = ({ size, ...other }) => {
```

Comment:
We may want to use the `forwardRef` API any time that we wrap a styled-component for re-export. This would allow someone to trigger a tooltip or something similar if necessary.